src/mame/drivers/ti85.cpp

```diff
@@ -750,6 +750,23 @@ static MACHINE_CONFIG_DERIVED( ti84p, ti83pse )
 	MCFG_DEVICE_REPLACE("flash", AMD_29F800T , 0)
 MACHINE_CONFIG_END
 
+static MACHINE_CONFIG_DERIVED( ti84pv2, ti83pse )
+	MCFG_DEVICE_MODIFY("membank1")
+	MCFG_DEVICE_PROGRAM_MAP(ti84p_banked_mem)
+
+	MCFG_DEVICE_MODIFY("membank2")
+	MCFG_DEVICE_PROGRAM_MAP(ti84p_banked_mem)
+
+	MCFG_DEVICE_MODIFY("membank3")
+	MCFG_DEVICE_PROGRAM_MAP(ti84p_banked_mem)
+
+	MCFG_DEVICE_MODIFY("membank4")
+	MCFG_DEVICE_PROGRAM_MAP(ti84p_banked_mem)
+
+	MCFG_MACHINE_START_OVERRIDE(ti85_state, ti84p )
+	MCFG_DEVICE_REPLACE("flash", AMD_29F800T , 0)
+MACHINE_CONFIG_END
+
```

Comment:
Why are you creating a new machine config that's a duplicate of an existing one?  There's no reason you can't use the same machine config with multiple systems.  The machine config name doesn't have to match the shortname of the system.